src/ua/hillel/Animals.java

```diff
@@ -0,0 +1,29 @@
+import java.util.Scanner;
+
+public class Animals {
+    int add_animals;
```

Comment:
Please rename it to `animalCounter` as it gives a clearer understanding of the purpose of this field.
Also please make this field private and static, because there is no point in exposing it outside + static is needed because you want to calculate only animals (not cats and dogs)!